### ISSUE TITLE
Can't save new questions after upgrading to v0.33.1

### ISSUE THREAD

**Issue by javiercr** [metabase/metabase#10805]
**Describe the bug**
After upgrading from v0.33.0 to v0.33.1 we can't create any more questions

**Logs**
```
09-05 09:00:35 DEBUG middleware.log :: POST /api/card 400 8.7 ms (0 DB calls)
{:errors {:result_metadata "value may be nil, or if non-nil, value must be an array of valid results column metadata maps."}}
```

**To Reproduce**
1. Go to 'Ask a question' > Simple question
2. Select any database and any table
3. Press save and then save again.

**Expected behavior**
Should save the question.

**Screenshots**
![image](https://user-images.githubusercontent.com/855995/64327822-a335c980-cfcc-11e9-826a-9593f3880565.png)

**Information about your Metabase Installation:**
- Using the metabase v0.33.1 Docker image on a kubernetes cluster
- Metabase internal database: H2

**Severity**
I think it's a major issue :)

**Comment by etrepat** [comment]
Same here after upgrading to v0.33.1 (via jar file). The error message appears translated (spanish) but it's the same one.

**Comment by CoconutTherapy** [comment]
Same here

**Comment by kritop** [comment]
same here

**Comment by GitsMcGee** [comment]
I'm also getting the same problem, but not just with new questions. I get it when trying to update any existing question I have. For example, I just received this message when I tried to add conditional formatting to a week-old report.

**Comment by sbelak** [comment]
We're already testing a fix in #10809

**Comment by mazameli** [comment]
Fixed by #10809

**Comment by javiercr** [comment]
Wow, that was quick, thank you! :clap:
